install/eks_eksctl/07_kubectl-apply.md

```diff
@@ -8,6 +8,7 @@
   - [How to check](#how-to-check)
 
 <!-- /TOC -->
+<!-- /TOC -->
```

Comment:
dont make unnecessary change

Reply:
Handled in [0ae93f4](https://github.com/ajktown/docs/pull/65/commits/0ae93f4a216a68affc30883ea5728bbc8364bcd2)